### PR TITLE
msgSend(), msgRecv(), msgRespond() functions documentation

### PR DESCRIPTION
**Pull request by niewim19**
JIRA: PD-234

### CODE REVIEW

File: libc/functions/m/msgrecv.phrtos.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+## Conformance
+Phoenix-RTOS specific
+## Description
+
+This function should read a message from _port_ and store its contents in `msg_t` structure pointed by _m_. _rid_ parameter specifies receiving context and should be passed to `msgRespond()`.
+
+Upon calling `msgRecv()` the receiving thread is suspended until one of the following occurs:
+ - a new message is received
+ - port is closed
+ - an error occurs
```
Comment (agkaminski) [inline libc/functions/m/msgrecv.phrtos.md:17]: Or signal is received

File: libc/functions/m/msgrecv.phrtos.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Return value
+
+If an error occurred during a function call an error value shall be returned. Otherwise returns `0`.
+
+## Errors
+
+This function shall fall if:
+
+ * `-EINVAL` - _port_ does not name an existing port, or _port_ is closed
```
Comment (agkaminski) [inline libc/functions/m/msgrecv.phrtos.md:33]: + -EINTR

File: libc/functions/m/msgsend.phrtos.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+If an error occurred during a function call an error value shall be returned. Otherwise it returns `0`.
+
+## Errors
+
+This function shall fall if:
+
+ * `-EINVAL` - _port_ does not name an existing port, or _port_ is closed.
+
+ * `-EINVAL` - _m_ points to an uninitialized message structure.
```
Comment (agkaminski) [inline libc/functions/m/msgsend.phrtos.md:30]:
```suggestion
 * `-EINVAL` - _port_ does not name an existing port or _port_ is closed or _m_ points to an uninitialized message structure.
```

File: libc/functions/m/msgsend.phrtos.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+This function may fall if:
+
+ * `-EINTR` - calling thread was woken up by signal before the message was sent.
```
Comment (agkaminski) [inline libc/functions/m/msgsend.phrtos.md:34]:
```suggestion
 * `-EINTR` - calling thread was woken up by signal before the message was received by the server.
```

File: libc/functions/m/msgsend.phrtos.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+## Return value
+
+If an error occurred during a function call an error value shall be returned. Otherwise it returns `0`.
+
+## Errors
+
+This function shall fall if:
+
+ * `-EINVAL` - _port_ does not name an existing port, or _port_ is closed _m_ points to an uninitialized message structure.
```
Comment (agkaminski) [inline libc/functions/m/msgsend.phrtos.md:28]:
```suggestion
 * `-EINVAL` - _port_ does not name an existing port, or _port_ is closed, or _m_ points to an uninitialized message structure.
```

Comment (niewim19) [reply libc/functions/m/msgsend.phrtos.md]: Sorry for that. Corrected.

**Review by agkaminski — APPROVED** [review]
LGTM

**Pull request merged**